code/modules/mob/mob_grab.dm

```diff
@@ -212,9 +212,94 @@
 				if(!do_mob(user, affecting)||!do_after(user, 100)) return
 			user.visible_message("<span class='danger'>[user] devours [affecting]!</span>")
 			affecting.loc = user
-			attacker.stomach_contents.Add(affecting)
+			choker.stomach_contents.Add(affecting)
 			qdel(src)
 
+////////////////////LING CODE HERE/////////////////////////////////////////////////
+		if( (ishuman(user) && (user.mind.changeling) && ishuman(affecting) ) )
+			var/mob/living/carbon/human/attacker = user
+			var/mob/living/carbon/human/victim = affecting
+			var/lingbite_armor_check = victim.run_armor_check(user.zone_sel.selecting, "melee")
+			var/obj/item/organ/limb/L = victim.get_organ(check_zone(user.zone_sel.selecting))
+			if(victim.health > 0)

```

Comment:
@Iamgoofball that's not how configs work.

@Man-Shroom either change it to @Incoming5643 's suggestion, or use the config value for crit's beginning.